test.json

```diff
@@ -0,0 +1 @@
+["transition: [instant, out]","hide_menu: [main]","hide_menu: [pause]","change_scene: [res://game/rooms/room02/room02.tscn, false]","set_global: [ANIMATION_RESOURCES, {}, true]","set_global: [ESC_CURRENT_SCENE, room2, true]","set_global: [ESC_LAST_SCENE, room1, true]","set_global: [FORCE_LAST_SCENE_NULL, False, true]","set_global: [dialog_advance, 0, true]","set_global: [dialog_popup_advance, 0, true]","set_global: [room1_visited, True, true]","set_active_if_exists: [_camera, True]","set_interactive: [_camera, True]","set_state: [_camera, default, true]","set_active_if_exists: [_music, True]","set_interactive: [_music, True]","set_state: [_music, res://game/sfx/contemplation.ogg, true]","play_snd: [res://game/sfx/contemplation.ogg, _music]","set_active_if_exists: [_sound, True]","set_interactive: [_sound, True]","set_state: [_sound, default, true]","stop_snd: [_sound]","set_active_if_exists: [_speech, True]","set_interactive: [_speech, True]","set_state: [_speech, default, true]","stop_snd: [_speech]","set_active_if_exists: [player, True]","set_interactive: [player, True]","set_state: [player, default, true]","teleport_pos: [player, 361, 496]","set_angle: [player, 201]","set_active_if_exists: [r2_bridge, True]","set_interactive: [r2_bridge, True]","set_state: [r2_bridge, default, true]","set_active_if_exists: [r2_button, True]","set_interactive: [r2_button, True]","set_state: [r2_button, default, true]","set_active_if_exists: [r2_button_right, True]","set_interactive: [r2_button_right, True]","set_state: [r2_button_right, default, true]","set_active_if_exists: [r2_l_exit, True]","set_interactive: [r2_l_exit, True]","set_state: [r2_l_exit, default, true]","set_active_if_exists: [r2_player_start, True]","set_interactive: [r2_player_start, True]","set_state: [r2_player_start, default, true]","set_active_if_exists: [r2_r_exit, True]","set_interactive: [r2_r_exit, True]","set_state: [r2_r_exit, default, true]","set_active_if_exists: [r2_right_platform, True]","set_interactive: [r2_right_platform, True]","set_state: [r2_right_platform, default, true]","transition: [instant, in]"]
```

Comment:
What's this file for?

Reply:
Oops that was a JSON export I did with a savegame. Deleting.